web/apps/labelstudio/src/pages/DataManager/DataManager.jsx

```diff
@@ -59,22 +60,22 @@ const buildLink = (path, params) => {
 };
 
 export const DataManagerPage = ({ ...props }) => {
+  const [dependencies] = useState(loadDependencies);
   const toast = useContext(ToastContext);
   const root = useRef();
   const params = useParams();
   const history = useHistory();
   const api = useAPI();
   const { project } = useProject();
-  const LabelStudio = useLibrary("lsf");
-  const DataManager = useLibrary("dm");
   const setContextProps = useContextProps();
   const [crashed, setCrashed] = useState(false);
+  const [loading, setLoading] = useState(!window.DataManager || !window.LabelStudio);
   const dataManagerRef = useRef();
   const projectId = project?.id;
 
   const init = useCallback(async () => {
-    if (!LabelStudio) return;
-    if (!DataManager) return;
+    if (!window.LabelStudio) return;
+    if (!window.DataManager) return;
```

Comment:
ideally we should sub these checks out for a check against `dependencies` this will help us move towards removing dependency on these window declarations of LabelStudio and DataManager

Reply:
alternatively we hold off and dig deeper into it on a follow up ticket

Reply:
This makes sense, we came up with the alternative approach later and this is an adage of a previous incarnation. We should address it here, and use this as the canonical approach for LSE as well.